templates/telemetry/mobile-event/mobile-event.1.schema.json

```diff
@@ -2,5 +2,11 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "type": "object",
   "title": "mobile-event",
-  @TELEMETRY_MOBILEEVENT_1_JSON@
+  "properties": {
+    @TELEMETRY_MOBILEEVENT_1_JSON@,
+    "processStartTimestamp": {
```

Comment:
I would instead move `processStartTime` to the `mobile-event` template (so it would be included with `focus-event` as well), include `properties` in that template, and have the only difference between `mobile-event` and `focus-event` JSON schemas be the `required` section (where `mobile-event` should have `processStartTimestamp`.

Reply:
Done.